server/services/store/mattermostauthlayer/mattermostauthlayer.go

```diff
@@ -205,7 +212,7 @@ func (s *MattermostAuthLayer) GetWorkspace(id string) (*model.Workspace, error)
 		}
 		var name string
 		if err := rows.Scan(&name); err != nil {
-			log.Fatal(err)
```

Comment:
This `log.Fatal` looks wrong here, I think this should just return an error.

Reply:
I agree.